test/controllers/api/v2/concerns/bulk_hosts_extensions_test.rb

```diff
@@ -15,10 +15,10 @@ def initialize(params = {})
 
   class Api::V2::BulkHostsExtensionsTest < ActiveSupport::TestCase
     def models
-      @host1 = FactoryBot.create(:host)
-      @host2 = FactoryBot.create(:host)
-      @host3 = FactoryBot.create(:host)
       @organization = get_organization
+      @host1 = FactoryBot.create(:host, organization: @organization)
```

Comment:
we have some fixtures for hosts and I think they have an organization already - try `hosts(:one)` for example and see if we can avoid creating these. if it's not a drop-in replacement then don't worry about it

Reply:
Thanks, updating.